lib/saml/user_attributes/ssoe.rb

```diff
@@ -184,7 +184,11 @@ def loa_highest
       end
 
       def multifactor
-        safe_attr('va_eauth_multifactor')&.downcase == 'true'
+        if csid == SAML::User::LOGINGOV_CSID
+          safe_attr('va_eauth_aal') >= '2'
```

Comment:
Can we make the `2` a constant? I believe we could add it in the `aal.rb` initializer

Reply:
Also, can we make it `==`? I just suggest that because anything greater than `2` would be unexpected behavior, so we may as well prevent it